mlir/utils/jenkins/miopen-tests/miopen_validate.sh

```diff
@@ -96,8 +96,9 @@ function create_configs() {
         if [[ $line == \#* || $line == "" ]]; then
             continue
         fi
-        echo "\"$DRIVER\"" "${FIXED_CONFIG_ARGS[@]}" "$line" >>"$TMPFILE"
+        echo "$DRIVER" "${FIXED_CONFIG_ARGS[@]}" "$line" >>"$TMPFILE"
```

Comment:
[Question] Is this a refactor? Looks like you get rid of the unecessary `\`

Reply:
... I'm still not sure if those might be needed, but we don't expect $DRIVER to contain spaces, so it's probably fine in practice